lib/dotcom_web/components/route_pills.ex

```diff
@@ -0,0 +1,63 @@
+defmodule DotcomWeb.Components.RoutePills do
+  @moduledoc """
+  Reusable components for displaying subway routes.
+  """
+
+  use DotcomWeb, :component
+
+  attr :route_id, :string
+
+  def route_pill_with_modifiers(assigns) do
+    ~H"""
+    <span class="flex">
+      <.route_pill route_id={@route_id} />
+      <span class="h-6 -ml-1 flex gap-0.5">
+        <.route_modifier :for={modifier_id <- @modifier_ids} modifier_id={modifier_id} />
```

Comment:
Was playing around with this and found that using the existing `DotcomWeb.Components.RouteSymbols.route_icon` component here looks great too:

```
<.route_icon :for={modifier_id <- @modifier_ids} route={%Routes.Route{id: modifier_id}} class="ring-2 ring-white rounded-full" />
```

Reply:
It looked funny to me in Firefox, because Firefox aligns the text in the route pills slightly differently from how it's aligned in the SVG's.

If we get SVG's for the route pills also, then I'm all in favor of using the existing `<.route_icon />` components, but I'm a bit leery of having the component be partially SVG and partially text-in-a-box.